### DISCUSSION TITLE
Problema con CheckBox Select ALL jquery 3.6

### DISCUSSION THREAD

**Discussion by rafaelg21** [jquery/jquery#5089] [General]
Buenos días saludos hice una actualización de JQuery 1.4.2 a 3.6.0
y el seleccionar todos los checkbox deja de funcionar cuando selecciono uno y luego le doy para seleccionar todos y desmarcar y este sigue marcado.

`<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-US" lang="en-US">
<head>
	<title>New document</title>
	<meta http-equiv="content-type" content="text/html; charset=iso-8859-1" />
	<script type="text/javascript" src="https://code.jquery.com/jquery-3.6.0.js"></script>
	<script language="javascript">
	 $(document).ready(function(){
	   $(".check_todos").click(function(event){
	     if($(this).is(":checked")) {
		 	$(".ck:checkbox:not(:checked)").attr("checked", "checked");
		 }else{
			 $(".ck:checkbox:checked").removeAttr("checked");
		 }
	   });
	 });
	</script>
</head>
<body>
<form>
<p><input name="Todos" type="checkbox" value="1" class="check_todos"/>Seleccionar todos</p>
<p>
  <input name="elemento1" type="checkbox" value="1" class="ck"/>elemento 1<br />
  <input name="elemento2" type="checkbox" value="2" class="ck"/>elemento 2<br />
  <input name="elemento3" type="checkbox" value="3" class="ck"/>elemento 3<br />
  <input name="elemento4" type="checkbox" value="4" class="ck"/>elemento 4<br />
  <input name="elemento5" type="checkbox" value="5" class="ck"/>elemento 5</p>
</form>

</body>
</html>`

**Comment by mgol** [comment]
This is a breaking change in jQuery 3.0.0; more info here: https://jquery.com/upgrade-guide/3.0/#breaking-change-removeattr-no-longer-sets-properties-to-false
